README.md

```diff
@@ -1,25 +1,38 @@
 # HOOMD-blue
 
-HOOMD-blue is a general purpose particle simulation toolkit. It performs hard particle Monte Carlo simulations
+[![Citing HOOMD](https://img.shields.io/badge/cite-hoomd-blue.svg)](https://glotzerlab.engin.umich.edu/hoomd-blue/citing.html)
+[![conda-forge](https://img.shields.io/conda/vn/conda-forge/hoomd.svg?style=flat)](https://anaconda.org/conda-forge/hoomd)
+[![docker](https://img.shields.io/badge/docker-glotzerlab/software-blue.svg)](https://hub.docker.com/r/glotzerlab/software)
+[![conda-forge Downloads](https://img.shields.io/conda/dn/conda-forge/hoomd.svg?style=flat)](https://anaconda.org/conda-forge/hoomd)
+[![CircleCI (all branches)](https://img.shields.io/circleci/project/github/glotzerlab/hoomd-blue.svg?style=flat)](https://circleci.com/gh/glotzerlab/hoomd-blue)
+[![Read the Docs](https://img.shields.io/readthedocs/hoomd-blue/stable.svg)](https://hoomd-blue.readthedocs.io/en/stable/?badge=stable)
+[![Contributors](https://img.shields.io/github/contributors/glotzerlab/hoomd-blue.svg?style=flat)](https://hoomd-blue.readthedocs.io/en/stable/credits.html)
+[![License](https://img.shields.io/badge/license-BSD--3--Clause-green.svg)](LICENSE)
+
+**HOOMD-blue** is a general purpose particle simulation toolkit.It performs hard particle Monte Carlo simulations
 of a variety of shape classes, and molecular dynamics simulations of particles with a range of pair, bond, angle,
-and other potentials. HOOMD-blue runs fast on NVIDIA GPUs, and can scale across
-many nodes. For more information, see the [HOOMD-blue website](http://glotzerlab.engin.umich.edu/hoomd-blue).
+and other potentials. **HOOMD-blue** runs fast on NVIDIA GPUs, and can scale across thousands of nodes.
+For more information, see the [**HOOMD-blue** website](https://glotzerlab.engin.umich.edu/hoomd-blue/).
 
-# Tutorial
+## Resources
 
-[Read the HOOMD-blue tutorial online](http://nbviewer.jupyter.org/github/joaander/hoomd-examples/blob/master/index.ipynb).
+- [Reference Documentation](https://hoomd-blue.readthedocs.io/):
+  Full package Python API, usage information, and feature reference.
+- [hoomd-users Google Group](https://groups.google.com/d/forum/hoomd-users):
+  Ask questions to the **HOOMD-blue** community.
+- [**HOOMD-blue** Tutorial](https://nbviewer.jupyter.org/github/glotzerlab/hoomd-examples/blob/master/index.ipynb):
+  Beginner's guide, code examples, and sample scripts.
+- [**HOOMD-blue** website](https://glotzerlab.engin.umich.edu/hoomd-blue/):
+  Additional information, benchmarks, and publications.
 
-## Installing HOOMD-blue
+## Installation
 
-**HOOMD-blue** binary images are available on [Docker Hub](https://hub.docker.com/) and packages on [conda-forge](https://conda-forge.org/).
+**HOOMD-blue** binaries are available via the
+[Docker image glotzerlab/software](https://hub.docker.com/r/glotzerlab/software)
```

Comment:
Refer to images more generically as the images are now available both on Docker Hub and [Singularity Hub](https://singularity-hub.org/collections/1663).